events: Mention events not in the default event type mask (fixes #404) (#741)
syncthing/docs: events/localchangedetected.rst

LocalChangeDetected
-------------------

Generated upon scan whenever the local disk has discovered an updated file from the
previous scan.  This does *not* include events that are discovered and copied from
other devices (:doc:`remotechangedetected`), only files that were changed on the
local filesystem.

> .. note:: This is not included in the default mask for the
>    :doc:`/rest/events-get` endpoint,but needs to be selected explicitly.

Reviewer: ```suggestion
.. note:: This event is not included in :doc:`/rest/events-get` endpoint without a mask specified, but needs to be selected explicitly.
```

I restructured the sentence, because there's no such thing as an explicit "default mask", you can just not give a mask at all - maybe that's a source of confusion? I am not sure about that part of the change. 
Repeating the "event" and space after comma is unrelated, and also applies to the other change below

Author: Right, it was probably getting too late when I made the change.
